src/sentry/api/base.py

```diff
@@ -95,6 +95,10 @@ class Endpoint(APIView):
 
     cursor_name = "cursor"
 
+    # Default Rate Limit Values, override in subclass
+    # Should be of format: { <http function>: { <category>: (limit, window) } }
+    rate_limits = {}
```

Comment:
I'm wondering if we should define a data class for some of this. So when we define a class, we do
```
rate_limits = {
    "GET": {RateLimitCategory.IP: RateLimit(500, 60)},
}
```
Might be a little nicer for type hinting, and also easier to figure out what those params are when reading code, since you can jump to the definition of `RateLimit`.